randovania/gui/lib/multiplayer_session_api.py

```diff
@@ -218,3 +285,6 @@ async def request_session_update(self):
             "multiplayer_request_session_update",
             self.current_session_id
         )
+
+    def upload_layout_description(self, layout: LayoutDescription):
+        pass
```

Comment:
?

Reply:
Ooops this became `prepare_to_upload_layout`